packages/server/src/sdk/app/rows/search/sqs.ts

```diff
@@ -164,8 +164,13 @@ export async function search(
         throw new Error("SQS cannot currently handle multiple queries")
       }
 
-      let sql = query.sql,
-        bindings = query.bindings
+      let sql = query.sql
+      let bindings = query.bindings?.map(b => {
+        if (typeof b === "boolean") {
+          return b ? 1 : 0
+        }
+        return b
+      })
```

Comment:
SQLite has no boolean type, and so the SQS API does not accept boolean bindings. So we convert them to 1s and 0s.

Reply:
I'm wondering if we should move this up a layer in some way - if we bake this into the SQL generation rather than the search API, it means if we use SQS anywhere else we don't have to worry about this.